src/Console/Command/FixCommand.php

```diff
@@ -100,6 +100,7 @@ public function getHelp(): string
 * `json` follows the `own JSON schema </doc/schemas/fix/schema.json>`_
 * `junit` follows the `JUnit XML schema from Jenkins </doc/schemas/fix/junit-10.xsd>`_
 * `xml` follows the `own XML schema </doc/schemas/fix/xml.xsd>`_
+* `gitlab` follows the `codeclimate JSON schema </doc/schemas/fix/codeclimate.json>`_
```

Comment:
Can `gitlab` be placed a few lines higher, so the list keeps its alphabetical order?